docs/mixins/state.md

```diff
@@ -1,41 +1,76 @@
-# Marionette.Toolkit.StateClass
+# StateMixin
 
-`Marionette.Toolkit.StateClass` is a `Marionette.Object` with a `Backbone.Model` for keeping state.
+`StateMixin` is a public mixin for [`App`](./../app.md) or any Marionette class definition that uses a `Backbone.Model` for keeping state.
```

Comment:
It'd be cool to list this as `Marionette.Toolkit.StateMixin` so people know how to get to it.